src/services/rhsm/rhsmConstants.js

```diff
@@ -326,19 +329,36 @@ const RHSM_API_QUERY_CATEGORY_TYPES = {
 const RHSM_API_QUERY_GRANULARITY_TYPES = RHSM_API_RESPONSE_GRANULARITY_TYPES;
 
 /**
- * ToDo: Clean up sort params once hosts is fully deprecated
- * These sort params are focused on the instances api NOT hosts. Since there
- * are minor differences, hosts sort params are maintained in /types/rhsmApiTypes.js as
- * we migrate towards hosts deprecation. Subscriptions sorts
+ * ToDo: Remove RHSM_API_QUERY_INVENTORY_HOSTS_SORT_TYPES once hosts is fully deprecated
+ * These sort params are focused on the hosts api.
  */
 /**
  * RHSM API query/search parameter SORT type values for HOSTS.
  *
- * @type {{CORES: string, STORAGE_GIBIBYTES: string, SOCKETS: string, INSTANCE_HOURS: string,
- *     TRANSFER_GIBIBYTES: string, BILLING_PROVIDER: string, CORE_SECONDS: string, LAST_SEEN: string, NAME: string}}
+ * @type {{CORES: string, CORE_HOURS: string, HARDWARE: string, INSTANCE_HOURS: string, SOCKETS: string, MEASUREMENT: string,
+ *     LAST_SEEN: string, NAME: string}}
+ */
+const RHSM_API_QUERY_INVENTORY_HOSTS_SORT_TYPES = {
+  CORES: 'cores',
+  CORE_HOURS: 'core_hours',
+  HARDWARE: 'hardware_type',
+  INSTANCE_HOURS: 'instance_hours',
+  LAST_SEEN: 'last_seen',
+  MEASUREMENT: 'measurement_type',
+  NAME: 'display_name',
+  SOCKETS: 'sockets'
+};
+
+/**
+ * RHSM API query/search parameter SORT type values for general inventory displays.
+ *
+ * @type {{CORES: string, CORE_HOURS: string, STORAGE_GIBIBYTES: string, SOCKETS: string, INSTANCE_HOURS: string,
+ *     TRANSFER_GIBIBYTES: string, BILLING_PROVIDER: string, CORE_SECONDS: string, STORAGE_GIBIBYTE_MONTHS: string,
+ *     LAST_SEEN: string, NAME: string}}
  */
 const RHSM_API_QUERY_INVENTORY_SORT_TYPES = {
   ...RHSM_API_PATH_METRIC_TYPES,
+  ...RHSM_API_QUERY_INVENTORY_HOSTS_SORT_TYPES,
```

Comment:
required:
confirm this isn't going to be an issue with the sort-types check. we may need to keep it separate because of the casing differences

Reply:
resolved with adding an additional hosts sorting hook and independent export from rhsmConstants